Slim/Player/Player.pm

```diff
@@ -296,23 +293,31 @@ sub power {
 
 		$controller->playerActive($client);
 
-		if (!$controller->isPlaying() && !$noplay) {
-			
-			if ($resumeOn =~ /Reset/) {
-				# reset playlist to start, but don't start the playback yet
-				$client->execute(["playlist","jump", 0, 1, 1]);
-			}
-			
-			if ($resumeOn =~ /Play/ && Slim::Player::Playlist::track($client)
-				&& $prefs->client($client)->get('playingAtPowerOff')) {
-				# play even if current playlist item is a remote url (bug 7426)
-				# but only if we were playing at power-off (bug 7061)
-				$client->execute(["play"]); # will resume if paused
-			}
-		}		
+		# Decide if we shall resume playing on power 
+		$client->resumeOnPower();
 	}
 }
 
+sub resumeOnPower {
+	my $client = shift;
+
+	if (!$client->controller->isPlaying()) {
```

Comment:
This formerly used the `noplay` parameter to `power()`. If the removal was intentional (what's the semantics of the parameter), it can be removed from the parameter list now.

Reply:
My bad